pkg/localize/locales/en/cmd/login.en.toml

```diff
@@ -113,7 +113,7 @@ one = 'Logged in successfully to {{.Host}}'
 [login.error.noRealmInURL]
 one = 'the authentication URL is missing a realm'
 
-[login.log.debug.sshLoginDetected]
+[login.log.info.sshLoginDetected]
```

Comment:
I feel like that should be a debug message as users _may_ experience issues logging in, but they also may not depending on whether they have a UI connected to their SSH session.

Reply:
agree!